components/dialog/test/dialog-ifrau-contents.visual-diff.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script>
+		const preferNative = (window.location.search.indexOf('preferNative=false') === -1);
+		window.D2L = { DialogMixin: { preferNative: preferNative } };
+	</script>
+	<script type="module">
+		import '../../typography/typography.js';
+		import '../../button/button.js';
+		import '../dialog.js';
+	</script>
+	<title>d2l-dialog-ifrau</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
This line can go away now that legacy-Edge isn't something we care about. I removed it from all existing files earlier, but you didn't have that change when you copied this file. :)

Reply:
Oh, nice! Yeah, I forgot to merge main before I started working on the visual diff tests so there were a few other things I also needed to nuke. I'll remove this and compare against the existing file in main to make sure I caught all of it.